source/administration/object-management/object-lifecycle-management.rst

```diff
@@ -119,6 +119,8 @@ MinIO has two specific default behaviors for versioned buckets:
 
   To expire delete markers when there are no remaining versions for that object, specify the :mc-cmd:`~mc ilm rule add --expire-delete-marker` option when creating the expiration rule.
 
+- To expire all versions of an object after a specified period of days, including the current version *and* all non-current versions, use the :mc-cmd:`~mc ilm rule add --expire-all-object-versions` flag with the :mc-cmd:`~mc ilm rule add --expire-days` flag.
```

Comment:
I _wonder_ if what we need to emphasize is that this results in the permanent deletion of that object. Spitballing

```suggestion

- To expire *all* versions of an object after a specified period of days, use the :mc-cmd:`~mc ilm rule add --expire-all-object-versions` flag with the :mc-cmd:`~mc ilm rule add --expire-days` flag. 
  This permits the permanent deletion of the object after the specified number of days pass.
```

Reply:
Am mostly stuck between "can all mean something other than all" and "users may reasonably not consider the current object version to be a version"

Reply:
Oh I guess we could do

```suggestion
- To **permanently expire/delete** an object and all its versions after a specified period of days, use the :mc-cmd:`~mc ilm rule add --expire-all-object-versions` flag with the :mc-cmd:`~mc ilm rule add --expire-days` flag.
```

Reply:
I'll take it.